deploy/os/controller.yaml

```diff
@@ -11,6 +11,8 @@ spec:
     metadata:
       labels:
         app: che-workspace-controller
+        app.kubernetes.io/name: devworkspace-controller
```

Comment:
1. It's pod template labels, does it makes sense to include them into deployment's labels as well?

2. Should not match labels be the same as template's labels? If I'm not mistaken it's a requirement from some K8s/OpenShift version.

Reply:
I moved labels to k8s recommended one.
Please review https://github.com/sleshchenko/che-workspace-operator/commit/489f89a1df3c854cdb6aa48aabfac29dbf53da44 and pull it if it's OK

And again reference to Recommended labels for k8s objects https://kubernetes.cn/docs/concepts/overview/working-with-objects/common-labels/

Reply:
Makes sense -- I only copied what was present on the web-terminal-operator side. I'll add labels as you suggest.

Reply:
Added PR with a minor fixup: app label in the aggregate edit/view roles changed to `devworkspace-controller` instead of `che-plugin-registry`.